### PR TITLE
Outdated links in the about page

### LINKED ISSUE #4771: Outdated links in the about page
In the about page (again),

there are links tied to contributors when TEAMMATES was hosted in Google code.

screen shot 2016-02-15 at 12 26 44

However, all links to their contributions now leads to this same <a href = "https://code.google.com/archive/p/teammatespes/"> page</a>.

screen shot 2016-02-15 at 12 27 04

Is it better to list their names only since the links does not provide any additional information?

### PR DESCRIPTION
**Pull request by yvsssantosh**
Fixes #4771 

Updated about.html and removed links which are inactive

### CODE REVIEW

File: src/main/webapp/about.html [hunk 1]
```diff
@@ -322,23 +322,18 @@ <h1 id="caption">About Us</h1>
                     <ol>
                     <div style="margin: 0 auto; padding: 0 0 15px;">[Multiple contributions]</div>
                         <li>Ang Ji Kai 
-                            [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=zi.kai88"
-                                target="_blank">bug reporting/fixing, enhancements</a>]
+
```
Comment (thyageshm) [inline src/main/webapp/about.html]: I think we can remove the empty line

**Comment by thyageshm** [comment]
@damithc should we indicate that these devs contributed to google code repo? Or specify that they are "early devs"? Or just leave it empty?
if we choose to indicate, we can do something like:
```
<li>Ang Nickolas
          [google-code repo]
</li>
```
 or 
```
<li>Ang Nickolas
          [early developer]
</li>
```

**Comment by damithc** [comment]
May be give the text as before, but no hyperlink?

**Comment by yvsssantosh** [comment]
Hi,
There are merge issues with this PR
So I've created a new PR #5047 

Kindly Review

Thank You

**Comment by thyageshm** [comment]
new PR created

**Pull request closed without merging**